assets/components/stripePopUpButton/stripePopUpButton.jsx

```diff
@@ -15,9 +15,7 @@ import {
 // ---- Types ----- //
 
 type PropTypes = {
-  stripeLoaded: boolean,
-  setupStripeCheckout: Function,
-  onStripeClick: Function,
+  dispatch: Function,
```

Comment:
Where is this dispatch field being populated?

Reply:
It is populated on line 46
"If you do not supply your own mapDispatchToProps function or object full of action creators, the default mapDispatchToProps implementation just injects dispatch into your component’s props." (https://github.com/reactjs/react-redux/blob/master/docs/api.md)